package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-d3-graph",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Please drop the changes here as this should be pushed within the release commit for easy tracking (e.g. [commit for release 2.5.0](https://github.com/danielcaldas/react-d3-graph/commit/b476ef5bfff01a15813a201d2f08d450d5e288ae))